device/core/src/blob_upload/file_upload_api.ts

```diff
@@ -3,7 +3,7 @@
 
 'use strict';
 
-import { endpoint, X509, SharedAccessSignature } from 'azure-iot-common';
+import { endpoint, AuthenticationProvider } from 'azure-iot-common';
 import { Http as DefaultHttpTransport } from 'azure-iot-http-base';
 import { UploadParams, FileUpload as FileUploadInterface } from './blob_upload_client';
```

Comment:
>FileUpload as FileUploadInterface  [](start = 23, length = 34)

nit: aargh.  aliases.

Reply:
unrelated to the change :)

---
In reply to: [157069967](https://github.com/Azure/azure-iot-sdk-node/pull/180#discussion_r157069967) [](ancestors = 157069967)